### PR TITLE
filtering users by name and account type while exporting

### PR DESCRIPTION
**Pull request by saga0p**
Now the users are getting filtered according to account type and name when we export users, rather than exporting all the accounts , only the filtered accounts are getting downloaded

### CODE REVIEW

**Comment by saga2405** [comment]
issue #313

**Comment by saga2405** [comment]
@llaske @NikhilM98 can you please review this request (#313 )

**Comment by NikhilM98** [comment]
There are several changes which are not relevant to the issue (comments, unwanted spaces). Can you please remove them?

Also, if you look at the git diff, you'll find some "end of line" errors, can you please get rid of them too?

**Comment by saga2405** [comment]
sure, I'll do that

**Comment by saga2405** [comment]
@NikhilM98 done

File: add-admin.sh [hunk 1]
```diff
@@ -2,4 +2,4 @@ if ! [ $# -eq "3" ] ; then echo "You should provide user, password and URL" ; ex
 newuser=$1
 newpassword=$2
 newobject="user=%7B%22name%22%3A%22${newuser}%22%2C%22role%22%3A%22admin%22%2C%22password%22%3A%22${newpassword}%22%2C%22language%22%3A%22en%22%7D"
-curl -k -i -H "Content-Type: application/x-www-form-urlencoded" -X POST -d "${newobject}" $3
+curl -k -i -H "Content-Type: application/x-www-form-urlencoded" -X POST -d "${newobject}" $3
```
Comment (llaske) [inline add-admin.sh:5]: Remove this change from the PR

File: docs/install.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ To run Sugarizer Server using Docker and Docker Compose:
 
 Install Docker Compose
 
-	curl -L "https://github.com/docker/compose/releases/download/1.8.1/docker-compose-$(uname -s)-$(uname -m)" > /usr/local/bin/docker-compose
+	curl -L "https://github.com/docker/compose/releases/download/1.8.1/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
```
Comment (llaske) [inline docs/install.md:17]: Remove this change from the PR

File: env/sugarizer.ini [hunk 3]
```diff
@@ -1,3 +1,4 @@
+version = 1.4.0
```
Comment (llaske) [inline env/sugarizer.ini:1]: Remove this change from the PR

File: package-lock.json [hunk 4]
```diff
@@ -1,12806 +1,8 @@
 {
 	"name": "sugarizer-server",
 	"version": "1.4.0",
-	"lockfileVersion": 2,
+	"lockfileVersion": 1,
```
Comment (llaske) [inline package-lock.json:4]: Remove changes in this file from the PR

**Review by llaske — CHANGES_REQUESTED** [review]
Good job. It works and it's useful.
BTW it's no longer possible to export every users which is require.
I suggest to add an option to do that in the combo to allow Students/Teacher/Admin + All.

See also my comments below and remove comments on every line of your change.

**Comment by llaske** [comment]
Closed due to lack of activity.

**Pull request closed without merging**